image-counter-lambda/artifact.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Don't think this should be part of this PR?

Reply:
Honestly no idea how it came back